xla/service/cpu/tests/onednn_matmul_test.cc

```diff
@@ -1574,6 +1584,103 @@ TEST_F(MatmulTest, BroadcastedAddAfterFusion) {
   )");
 }
 
+// Test MM + BiasAdd + Add fusion : F32
+TEST_F(MatmulTest, SimpleTestF32WithBiasAndAddFusion) {
+  const char* matmul_module_str = R"(
+  HloModule matmul.bias.add.test.f32
+  ENTRY matmul.bias.add.test.f32 {
+    arg0.1 = f32[32,32,40,30] parameter(0), parameter_replication={false}
+    arg0.2 = f32[32,32,30,40]parameter(1), parameter_replication={false}
+    dot.7 = f32[32,32,40,40] dot(arg0.1, arg0.2), lhs_batch_dims={0,1}, lhs_contracting_dims={3}, rhs_batch_dims={0,1}, rhs_contracting_dims={2}
+    const.0 = f32[40] constant(15)
+    bcast.1 = f32[32,32,40,40] broadcast(const.0), dimensions={3}
+    add.0 = f32[32,32,40,40] add(dot.7,bcast.1)
+    const.1 = f32[32,32,40,40] constant(0.65)
+    add.1 = f32[32,32,40,40] add(add.0, const.1)
+    tuple.12 = (f32[32,32,40,40]) tuple(add.1)
+    ROOT get-tuple-element.13 = f32[32,32,40,40] get-tuple-element(tuple.12), index=0
+  })";
```

Comment:
This module is used 3 times for F32, B16, and F16. Could you please create a helper function that calls `absl::StrReplaceAll` based on the input dtype? E.g., for bf16 we could call `CreateMatmulBiasAddAndAddModuleText("f32", "bf16")` and in the function we have a template string that supports at most 2 data types:
```
    ...
    arg0.1 = ${dtype1}[32,32,40,30] parameter(0), parameter_replication={false}
    convert.0 = ${dtype2}[32,32,40,30] convert(arg0.1)
    ...
```
etc.

In the long run, I think we should set up parameterized tests, but we can do that later. I can create an example in this file after this PR is in.